scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -4156,11 +4159,16 @@ def run(self):
             self.__report_k8s_metrics = False
 
         global_log.info(
-            "kubernetes_monitor parameters: ignoring namespaces: %s, report_controllers: %s, report_metrics: %s"
+            (
+                "kubernetes_monitor parameters: ignoring namespaces: %s, report_controllers: %s, "
+                "report_metrics: %s, report_k8s_metrics: %s, log_mode: %s"
```

Comment:
I added new values to the end, just in case someone has custom parser defined for that log line (highly unlikely).

Reply:
If we are going to centralize this stuff, it might be a good idea to also record here if we are using docker or CRI to get a list of containers.

The line I was referring to is at: https://github.com/scalyr/scalyr-agent-2/blob/master/scalyr_agent/builtin_monitors/kubernetes_monitor.py#L2649

Essentially, if we aren't in syslog mode, then there are still two different ways we get containers.  Either using Docker or using a filesystem based CRI method.  If we aren't using Docker, than I don't think we'll get metrics.

Reply:
Thanks, will update it to also log that.

Reply:
Pushed a change so we now also log that value.